src/upgrade.js

```diff
@@ -52,8 +52,8 @@ available updates.\
     getInstalledPackages(options) {
       let packages = [];
       for (let name of fs.list(this.atomPackagesDirectory)) {
-        var pack;
-        if (pack = this.getIntalledPackage(name)) {
+        let pack = this.getInstalledPackage(name);
```

Comment:
You're probably not going to believe this, but the actual function name in this file has been `getIntalledPackage` (note the typo, a missing "s" in "intalled") this whole time, since 2014 (https://github.com/atom/apm/commit/c8d61000#diff-0aa64babcef87ba238fed061035f2fe14c587092a7339b55c5e4033c37370d70R46).

(Regarding the singular `getIntalledPackage`, not to be confused with the plural `getInstalledPackages` ("Package**s**") which has/had no typo.)

So, we can either revert the spelling here to the previous (typoed but defined) function name here, or rename the function elsewhere in the file that this line calls, so as to not have a typo in its name, lol.

But yeah, this broke tests by fixing a typo, I honestly did not see that coming.

Thanks again for working on this! Nice to see readability improvements to the code!!!!!! We sure need it, as debugging this PR has helped prove, IMO!!! Thank you!